packages/yoastseo/src/languageProcessing/helpers/sentence/getSentences.js

```diff
@@ -49,7 +48,6 @@ export default function( text, memoizedTokenizer ) {
 	// Remove Table of Contents.
 	text = excludeTableOfContentsTag( text );
 	// Unify only non-breaking spaces and not the other whitespaces since a whitespace could signify a sentence break or a new line.
-	text = unifyNonBreakingSpace( text );
 
```

Comment:
Lines removed were introduced by this PR: https://github.com/Yoast/wordpress-seo/pull/18416
Code was removed because with our changes it seems unnecessary. Removing it did not affect behaviour of assessments, so it was deemed safe.